packages/mds-db/geographies.ts

```diff
@@ -34,9 +34,10 @@ export async function readGeographies(params: Partial<ReadGeographiesParams> = {
   try {
     const client = await getReadOnlyClient()
 
-    const { get_published, get_unpublished, geography_ids } = {
+    const { get_published, get_unpublished, geography_ids, only_published_after } = {
       get_published: false,
       get_unpublished: false,
+      only_published_after: null,
       ...params
     }
```

Comment:
This can also be done as follows:
```ts
const { get_published = false, get_unpublished = false, geography_ids, only_published_after = null } = params
```
Also, the `only_published_after = null` can probably just default to undefined based on usage.

Reply:
removed!